src/components/common/SignInDialog.vue

```diff
@@ -62,6 +62,11 @@ export default defineComponent({
     const oauthStore = useOauthStore();
 
     const gateways = [
+      {
+        id: BnetOAuthRegion.us,
+        name: t("gatewayNames.America"),
+        uri: "https://battle.net",
```

Comment:
Maybe the correct link is:

```suggestion
        uri: "https://us.battle.net",
```

I learned that from this comment: https://github.com/w3champions/launcher/pull/77#issuecomment-765263822.